### PR TITLE
Update the download link to grafana dashboard

### PR DESCRIPTION
**Pull request by simondeziel**
https://grafana.com/orgs/lxd/dashboards/19131 still doesn't work but the API version works.

### CODE REVIEW

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-          sudo snap refresh lxd --channel=latest
+          sudo snap refresh lxd
```
Comment (tomponline) [inline .github/workflows/builds.yml:22]: Should we be using latest/edge here for testing against that?

**Review by tomponline — APPROVED** [review]

File: .github/workflows/builds.yml (hunk 1 above)
Comment (simondeziel) [reply .github/workflows/builds.yml:22]: The builds job uses what's available from the base OS so LXD 5.0 here. The tests job uses the `latest/stable` channel so I think that's the best of both worlds.

Comment (simondeziel) [reply .github/workflows/builds.yml:22]: The tests job should test the charm itself so using `latest` was deemed good enough. Considering how reliable `latest/edge` is, it shouldn't be too bad to switch.

The builds job doesn't even exercise LXD nor the charm directly, it's merely having `charmcraft` use LXD to standup build environment, same as what `snapcraft` does. As such, sticking to the OS provided snap means we can proceed with the actual build more quickly.

FYI, the charm is built during the tests too so we essentially build it twice, once with LXD 5.0 and another with 5.X.

Comment (tomponline) [reply .github/workflows/builds.yml:22]: OK makes sense thanks for explaining.

**Pull request merged**